### PR TITLE
Update validation and formatting scripts

### PR DESCRIPTION
**Pull request by Trikolon**
* Update npm script names
 * Add a test script which runs all the tests
 * Apply prettier to all files in the repository
 * Improve logging in validateRules.
 * Remove `last_modified` and `schema` fields

### CODE REVIEW

File: test/validateRules.js [hunk 1]
```diff
@@ -68,17 +73,21 @@ const ajv = new Ajv({ loadSchema });
   ruleList.data.forEach((rule) => {
```
Comment (pdehaan) [inline test/validateRules.js:73]: FWIW, you can also define the loop index, `i`, in [`Array#forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) and JavaScript will tell you the index (versus tracking it separately and having to increment it on L86 below).

```js
ruleList.data.forEach((rule, i) => {
```

Comment (Trikolon) [reply test/validateRules.js:73]: Wow, how did I not know this. Thanks!

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -22,5 +21,5 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
-      - run: npm run validate
-      - run: npm run prettier
+      - run: npm run test-formatting
+      - run: npm run test-validate-rules
```
Comment (pdehaan) [inline .github/workflows/test.yml:25]: TIL: https://docs.npmjs.com/cli/v6/commands/npm-install-ci-test

Not sure if this is _too_ magic, but <kbd>npm cit</kbd> looks neat:
> _"This command runs <kbd>npm ci</kbd> followed immediately by <kbd>npm test</kbd>."_

Where <kbd>npm test</kbd> is currently an alias for:
```js
"test": "npm run test-formatting && npm run test-validate-rules",
```

Comment (Trikolon) [reply .github/workflows/test.yml:25]: Neat! However I prefer to keep them as separate runs in CI then it's easier to see which one failed.

**Review by artines1 — APPROVED** [review]
Thanks for improving the script here. I think that reporting clear error messages helps people to know what's wrong in the list.

**Pull request merged**